cfme/cloud/volume.py

```diff
@@ -21,8 +31,12 @@ def check_version():
         raise DestinationNotFound('Cloud Volumes does not exist in CFME 5.7+')
 
 
-class Volume(Navigatable):
+def get_volume_name():
+    """Returns the name of volume which page is opened"""
+    return sel.text('//h1').split()[0]
```

Comment:
could be something like this: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/openstack/test_host_role_type.py#L28